packages/starknet-snap/src/ui/components/ExecuteTxnUI.tsx

```diff
@@ -0,0 +1,168 @@
+import type { SnapComponent } from '@metamask/snaps-sdk/jsx';
+import {
+  Box,
+  Container,
+  Row,
+  Section,
+  Value,
+  Text,
+  Icon,
+} from '@metamask/snaps-sdk/jsx';
+import { formatUnits } from 'ethers/lib/utils';
+
+import type { FeeToken } from '../../types/snapApi';
+import { DEFAULT_DECIMAL_PLACES } from '../../utils/constants';
+import { AddressUI, JsonDataUI, RowUI } from '../fragments';
+import { FeeTokenSelector } from '../fragments/FeeTokenSelector';
+
+/**
+ * The form errors.
+ *
+ * @property to - The error for the receiving address.
+ * @property amount - The error for the amount.
+ * @property fees - The error for the fees.
+ */
+export type ExecuteTxnUIErrors = {
+  to?: string;
+  amount?: string;
+  fees?: string;
+};
+
+export type ExecuteTxnUIProps = {
+  type: string;
+  signer: string;
+  chainId: string;
+  maxFee: string;
+  calls: any[];
+  feeToken: string;
```

Comment:
```suggestion
  selectedFeeToken?: string;
```
it should be optional with default value ETH